security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogAction.xml

```diff
@@ -24,19 +24,27 @@
         <help>Pre-defined commands for this restart action.</help>
     </field>
     <field>
-        <label>Optional Parameters</label>
+        <label>Required Parameters</label>
         <type>header</type>
+        <style>method_table method_table_configd</style>
     </field>
     <field>
         <id>action.configd</id>
         <label>System Command</label>
         <type>dropdown</type>
         <help>Select a pre-defined system command which should be run for this action.</help>
+        <style>table_optional table_optional_configd</style>
+    </field>
+    <field>
+        <label>Required Parameters</label>
+        <type>header</type>
+        <style>method_table method_table_custom</style>
     </field>
     <field>
         <id>action.custom</id>
```

Comment:
this: https://github.com/fraenki/plugins/blob/6fcc7f8a5ae706279553601e606ee6d6da2b6766/security/acme-client/src/opnsense/mvc/app/views/OPNsense/AcmeClient/actions.volt#L59-L60